pkg/validation/validate_test.go

```diff
@@ -252,7 +252,7 @@ func TestValidateProfile(t *testing.T) {
 			MockLimits{
 				MaxProfileStacktraceSampleLabelsValue: 2,
 			},
-			NewErrorf(InvalidProfile, ProfileTooManyLabelsErrorMsg, `{foo="bar"}`, 3, 2),
+			NewErrorf(SampleLabelsLimit, ProfileTooManyLabelsErrorMsg, `{foo="bar"}`, 3, 2),
```

Comment:
We probably should call the variable `ProfileTooManySampleLabelsErrorMsg`, to make it clear which labels